examples/montecarlo.jl

```diff
@@ -24,7 +24,8 @@ function montecarlo(mc_eval::Function, mc_monitor::Function, reps, n_returns, po
     end
 
     if mod(pernode,pooled) != 0
-        pooled = div(pernode, commsize-1)
+        pooled = Int64(div(pernode, commsize-1))
```

Comment:
At this point, `pernode` is a `Float64`, since it is obtained via a division `/`.

The code would be cleaner if the calculation of `pernode` was changed to explicitly round up and be integer.

Also, some of the arguments to `montecarlo` itself should probably be restricted to `Integer`.